### DISCUSSION TITLE
Prosody Transfer/Cloning

### DISCUSSION THREAD

**Discussion by satani99** [coqui-ai/TTS#2805] [General Q&A] [answered]
Is there any solution for prosody transfer/cloning from one speaker to another?
i.e. Input: Reference Speaker, Target Speaker 
Output: Target speaker speaking exactly the same things with same voice as before but with prosody/style of reference speaker

**Comment by erogol** [comment, marked as answer]
There are ways with models that are explicitly using pitch like FastPitch, FastSpeech. However, it is not exposed in the API. You need to invent the wheel.
